src/Windows/R/Components/Impl/ContainerManager/Implementation/View/ContainerManagerControl.xaml.cs

```diff
@@ -42,12 +42,19 @@ public partial class ContainerManagerControl {
 
         private void ButtonFromFile_Click(object sender, RoutedEventArgs e) => ViewModel.ShowLocalDockerFromFile();
 
+        private void ButtonShowWorkspaces_Click(object sender, RoutedEventArgs e) => ViewModel.ShowConnections();
+
         private void Container_PreviewKeyUp(object sender, KeyEventArgs keyEventArgs) {
             if (keyEventArgs.Key == Key.Delete && !(keyEventArgs.OriginalSource is TextBox)) {
                 ViewModel.DeleteAsync(GetContainer(keyEventArgs)).DoNotWait();
             }
         }
 
+        private void Container_MouseDoubleClick(object sender, MouseButtonEventArgs e) {
```

Comment:
Enter/Space probably should do the same

Reply:
You can't navigate to text itself with keyboard, only buttons can be selected. Workspaces window has similar behavior.